docs/6.x/pack.md

```diff
@@ -117,6 +117,7 @@ The command below assumes that `build.sh` is located in the same working directo
 docker run \
        -v /tmp/tele-cache:/mnt/tele-cache \
        -v /var/run/docker.sock:/var/run/docker.sock \
+       -v $(which docker):/usr/bin/docker \
```

Comment:
I think running docker in the proper container would be more correct: https://docs.docker.com/engine/security/rootless/#rootless-docker-in-docker

Reply:
We just need the client binary here because we're not actually running docker in docker, we're running sibling containers. The various `docker:x` images aren't suitable for this because they don't have the interpreter that tele needs.  And if we're going to hack up an image, I'd rather install docker on top of one of our `debian-*` images than add the necessary tele runtime to one of  the `docker:x` images.